src/components/ReviewPageV2/components/PopularReviews/components/Top5Reviews/Top5Reviews.tsx

```diff
@@ -16,37 +18,55 @@ const Container = styled.div`
 `;
 
 // TODO: replace mock data that represents data from backend with real axois http request --------------
-type Props = {
-  title: string;
-  commentNum: number;
-};
+// type Props = {
+//   title: string;
+//   commentNum: number;
+// };
+
+// interface IReviewCardPros {
+//   review: IPopularReview;
+// }
 
-const mockTop5Reivews: Props[] = [
-  { title: 'Last of us review ', commentNum: 11 },
-  { title: 'Metroid prime review', commentNum: 22 },
-  { title: 'Elder ring review', commentNum: 33 },
-  { title: 'Zelda:breadth of the wild review22222', commentNum: 44 },
-  { title: 'Red dead redemption review', commentNum: 55 },
-];
-// TODO: -----------------------------------------------------------------------------------
+// const mockTop5Reivews: Props[] = [
+//   { title: 'Last of us review ', commentNum: 11 },
+//   { title: 'Metroid prime review', commentNum: 22 },
+//   { title: 'Elder ring review', commentNum: 33 },
+//   { title: 'Zelda:breadth of the wild review22222', commentNum: 44 },
+//   { title: 'Red dead redemption review', commentNum: 55 },
+// ];
 
 const Top5Reviews = () => {
-  const [top5Reviews, setTop5Reviews] = useState(mockTop5Reivews);
+  const [popularReviews, setPopularReviews] = useState<IPopularReview[]>();
+  const [isLoading, setIsLoading] = useState(true);
+  const [isError, setIsError] = useState(false);
 
+  useEffect(() => {
+    getPopularReviews()
+      .then((res) => {
+        const reviewsList = res.data;
+        console.log('popular: ', reviewsList);
+        setPopularReviews(reviewsList.data);
```

Comment:
上面  res.data.data这里就可以改成
```suggestion
        setPopularReviews(reviewsList);
```